jwst/assign_wcs/nircam.py

```diff
@@ -240,10 +240,10 @@ def tsgrism(input_model, reference_files):
     ycenter = Const1D(yc)
     ycenter.inverse = Const1D(yc)
 
-    setra = Const1D(input_model.meta.wcsinfo.crval1)
```

Comment:
Given that we supposedly removed FITS WCS keywords from spectroscopic products quite a while ago already, how was this working? `crval1` and `crval2` should not have been in the meta info.

Reply:
Good question. I think this is a subtle and very nasty bug due to the way modeling and datamodels work.
`input_model.meta.wcsinfo.crval1` returns `None` because crval1 is not set but is in the schema.
`modeling.Const1D` has a default parameter value (amplitude=1) which is set if `None` is passed.
The effect is that `setra` and `setdec` are set to 1 and the step works without crashing.